### PR TITLE
Added Google IO 2017 to the live site

### PR DESCRIPTION
**Pull request by Princu7**
Partially Fixes the issue #1456 

Changes:
* Adds Google IO sample event to the live site

**Test Website**
https://princu7.github.io/open-event-webapp

@aayusharora @geekyd @sumedh123 @harshitagupta30 Please review. Thanks :)

### CODE REVIEW

File: src/backend/overviewSite/index.html [hunk 1]
```diff
@@ -100,6 +100,12 @@ <h2 class="margin"> Apps with expandable sessions page </h2>
             <img src="./fa16small.jpg" class=""  alt="Image">
           </a>
         </div>
+        <div class="col-md-6 col-sm-6 col-xs-12 margin">
+          <p><strong>GoogleIO 2017<strong></p>
+          <a href='./GoogleIO2017/index.html' target='_blank'>
+            <img src="./googleIO.jpg" class=""  alt="Image">
```
Comment (sumedh123) [inline src/backend/overviewSite/index.html]: For the alt attribute, keep it as "backgroungImage"

Comment (aayusharora) [inline src/backend/overviewSite/index.html]: can you compress this ?

Comment (Princu7) [reply src/backend/overviewSite/index.html]: Sure. Thanks!

**Comment by Princu7** [comment]
**Test Website**
http://princu7.github.io/open-event-webapp

**Pull request merged**